lib/geoengineer/resources/aws_security_group.rb

```diff
@@ -26,6 +27,13 @@ def flatten_cidr_and_sg_blocks
     end
   end
 
+  def uniq_cidr_and_sg_blocks
+    (self.all_ingress + self.all_egress).each do |in_eg|
+      in_eg.cidr_blocks      = in_eg.cidr_blocks.uniq     if in_eg.cidr_blocks
+      in_eg.security_groups  = in_eg.security_groups.uniq if in_eg.security_groups
+    end
+  end
```

Comment:
Could combine with `flatten_cidr_and_sg_blocks`, but this will do the trick just dandy.